tests/ci/setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# shellcheck disable=SC2120
+setup_circle() {
+  echo "=====> setup_circle on CIRCLE_NODE_INDEX: $CIRCLE_NODE_INDEX"
+  sudo -E CI=true make -e sshcommand
+  # need to add the dokku user to the docker group
+  sudo usermod -G docker dokku
+  [[ "$1" == "buildstack" ]] && BUILD_STACK=true make -e stack
+  sudo -E CI=true make -e install
+  sudo -E make -e setup-deploy-tests
+  bash --version
+  docker version
+  lsb_release -a
+  # setup .dokkurc
```

Comment:
Wait does dokku now require this directory? I thought it was optional?

Reply:
This has always been there. We place some CI specific settings there in `.circleci/config.yml`